bin/deploy.sh

```diff
@@ -106,7 +106,8 @@ chk_config()
     fi
     echo -e "=>\tComponents: \"${COMPONENTS[*]}\""
 
-    AM_URL="${URL_PREFIX:-openam}.${NAMESPACE}.${DOMAIN}" 
+    AM_URL="${URL_PREFIX:-openam}.${NAMESPACE}.${DOMAIN}"
+    IDM_URL="${IDM_URL_PREFIX:-openidm}.${NAMESPACE}.${DOMAIN}"
```

Comment:
looks good. This might shortly change in the future to a new 'login' prefix - but good for now